src/styles.scss

```diff
@@ -4,10 +4,10 @@
 
 @include mat.core();
 
-$my-primary: mat.define-palette(mat.$indigo-palette, 500);
-$my-accent: mat.define-palette(mat.$pink-palette, A200, A100, A400);
+$my-primary: mat.m2-define-palette(mat.$m2-indigo-palette, 500);
+$my-accent: mat.m2-define-palette(mat.$m2-pink-palette, A200, A100, A400);
 
-$my-theme: mat.define-light-theme(
+$my-theme: mat.m2-define-light-theme(
```

Comment:
Not a requirement but if you feel like making a follow up PR to use latest Angular Material rather than the deprecated m2, that'd be amazing:
- https://material.angular.io/guide/material-2-theming
- https://material.angular.io/guide/material-2-theming#how-to-migrate-an-app-from-material-2-to-material-3

As I said, not a requirement though and this MR will definitely be merged first!

Reply:
At my company we had a quick look at migrating to Material 3, but as far as I heard, it was quite a challenge. As the theming seems to be only used in the demo application, I kept it on material 2.

Reply:
Yup, fair enough